src/condition.cpp

```diff
@@ -266,7 +266,7 @@ bool Condition::isPersistent() const
 		return false;
 	}
 
-	if (!(id == CONDITIONID_DEFAULT || id == CONDITIONID_COMBAT)) {
+	if (!(id == CONDITIONID_DEFAULT || id == CONDITIONID_COMBAT || condition->getType() == CONDITION_MUTED)) {
```

Comment:
`condition->getType()`
to
`conditionType`

Reply:
I'm sorry, I did fast change and didnt notice.